frontend/js/src/user/Listens.tsx

```diff
@@ -254,7 +254,7 @@ export default class Listens extends React.Component<
         newAlert(
           "danger",
           "We could not load data for the now playing listen",
-          typeof error === "object" ? error.message : error
+          error.status ?? error.toString()
```

Comment:
I don't think this is a good idea the way it is implemented. There could be an existing `error.message` on some errors, which is a lot more informative than just the status number (which doesn't mean anything to end users)

Reply:
Most of the times error.message and error.toString() produce similar results so maybe replacing it with error.message ?? error.status should do

Reply:
Is it possible to give me some examples of those errors, so I can come up with solutions keeping them in mind and also test accordingly.

Reply:
The call to the API method `getFeedbackForUserForRecordings` in `loadFeedbackForNowPlaying` that you are catching passes through this utility to check requests: https://github.com/metabrainz/listenbrainz-server/blob/master/frontend/js/src/utils/APIService.ts#L489-L512